### ISSUE TITLE
Dashboard does not show trial images in the Preferential Optimization Tutorial

### ISSUE THREAD

**Issue by benjaminum** [optuna/optuna-dashboard#620]
### Description

The dashboard does not show the trials after following Preferential Optimization Tutorial.

The dashboard just shows an empty page. The expectation would be to see the color images as shown in the tutorial.

<img width="890" alt="image" src="https://github.com/optuna/optuna-dashboard/assets/217089/74932990-31b6-48a2-8a77-9b573ac97caf">

### How to Reproduce

The procedure to reproduce follows the tutorial here https://optuna-dashboard.readthedocs.io/en/latest/tutorials/preferential-optimization.html

For convenience here are the steps I used in bash

```bash
conda create -n optuna python=3.10
conda activate optuna
pip install "optuna>=3.3.0" "optuna-dashboard>=0.13.0b1" pillow botorch

git clone https://github.com/optuna/optuna-dashboard.git
cd optuna-dashboard/examples/preferential-optimization
python generator.py &
optuna-dashboard sqlite:///example.db --artifact-dir ./artifact
```

### Python version

3.10.13

### Optuna version

3.3.0

### optuna-dashboard version or git revision

0.13.0b1

### Web browser

Google Chrome, Firefox

**Comment by c-bata** [comment]
@benjaminum, thank you for trying out our beta release. We've just published the stable release v0.13.0. Could you please try it out and let me know if there are still any issues?
https://medium.com/optuna/announcing-optuna-3-4-0087644c92fa

**Comment by benjaminum** [comment]
@c-bata Thanks, the stable release works
